src/pages/LandingPage/Index.tsx

```diff
@@ -17,6 +17,10 @@ import evmTokensAbi from '../../abi/evmTokensAbi.json';
 import { Transaction } from "@mysten/sui/transactions";
 import useSubmitTransaction from "../../api/hooks/useSubmitTransaction";
 
+import { ipAddress, next } from '@vercel/edge'
+import { Ratelimit } from '@upstash/ratelimit'
+import { kv } from '@vercel/kv'
+
 
 const aptosFaucetAddress = '0x275f508689de8756169d1ee02d889c777de1cebda3a7bbcce63ba8a27c563c6f';
```

Comment:
Why are these here? This should just be calling the faucet upstream. If these are for testing, leave a comment and also make these reusable.

Reply:
these are used for rate limiting